### PR TITLE
Logbook widget in app inspector

### PR DESCRIPTION
**Pull request by algairim**
Logbook widget has been enhanced with multi-phrase search. `searchId` is now a minimum phrase to match with, if supplied in the directive arguments.
Screenshot 2021-07-19 at 18 38 50

### CODE REVIEW

**Comment by algairim** [comment]
Couple of UX questions:
* Do we want to preform an initial single query or start auto-query with default search parameter automatically (It is no at the moment)?
* Would it be better to add activity ID as part of logbook header, so that it will be explicit and imply that logbook is for this activity ID only, meaning some filter applied already. Like the following:
Screenshot 2021-07-20 at 09 24 56

**Comment by algairim** [comment]
Depends on https://github.com/apache/brooklyn-server/pull/1208

**Comment by algairim** [comment]
Added logbook specific in parentheses.
Screenshot 2021-07-20 at 14 32 58

**Comment by algairim** [comment]
Same for activities:
Screenshot 2021-07-20 at 14 38 23

File: ui-modules/utils/logbook/logbook.js [hunk 1]
```diff
@@ -91,8 +91,6 @@ export function logbook() {
         $scope.fieldsToShow = ['timestamp', 'class', 'message']
         $scope.logFields = [
             {name: 'Timestamp',   value: 'timestamp',  selected: true},
-            {name: 'Task ID',     value: 'taskId',     selected: false},
-            {name: 'Entity IDs',  value: 'entityIds',  selected: false},
```
Comment (algairim) [inline ui-modules/utils/logbook/logbook.js:90]: @jcabrerizo, there is no task ID or entity ID in logs.

Comment (jcabrerizo) [reply ui-modules/utils/logbook/logbook.js:90]: Have you tried to invoke an effector? Is how I teste it and the taks Id was there

Comment (algairim) [reply ui-modules/utils/logbook/logbook.js:90]: Ok, I see now, reverting back.
Screenshot 2021-07-20 at 19 19 59

**Review by jcabrerizo — APPROVED** [review]
LGTM @algairim

**Pull request merged**